tests/benchmarks/timing_info/testcase.yaml

```diff
@@ -1,4 +1,9 @@
 tests:
-  benchmark.timing:
-    arch_whitelist: x86 arm
+  benchmark.timing.default_kernel:
+    arch_whitelist: x86 arm arc
```

Comment:
shouldn't this whitelist more architectures, actually should run all?

Reply:
yeah the remaining architectures should come here. I missed it